src/portable/renesas/usba/dcd_usba.c

```diff
@@ -245,12 +264,6 @@ static int fifo_write(volatile void *fifo, pipe_state_t* pipe, unsigned mps)
 
   hw_fifo_t *reg = (hw_fifo_t*)fifo;
   uintptr_t addr = pipe->addr + pipe->length - rem;
-  if (addr & 1u) {
```

Comment:
seems like a change in logic, I don't understand the mcu enough to decide if modification is correct. Maybe if you could, explain/confirm this a bit.

Reply:
I think this change is correct.

The codes are regarding load from misaligned address. At that time I thought that RX MCU has no support for load from misaligned address. But, it was my misunderstanding. In fact, it has support for load from misaligned address. So, the part is unnecessary.

Reply:
thank you very much for confirmation :+1: :+1: